packages/gatsby-theme-api-docs/package.json

```diff
@@ -24,18 +24,18 @@
     "@commercetools-docs/gatsby-transformer-mdx-introspection": "15.0.4",
     "@commercetools-docs/gatsby-transformer-raml": "13.1.1",
     "@commercetools-docs/ui-kit": "19.3.0",
-    "@commercetools-uikit/design-system": "^15.0.0",
-    "@commercetools-uikit/spacings-inline": "^15.0.0",
-    "@commercetools-uikit/spacings-stack": "^15.0.0",
-    "@emotion/react": "^11.7.1",
-    "@emotion/styled": "^11.6.0",
-    "gatsby-source-filesystem": "4.13.0",
+    "@commercetools-uikit/design-system": "15.2.4",
+    "@commercetools-uikit/spacings-inline": "15.2.4",
+    "@commercetools-uikit/spacings-stack": "15.2.4",
+    "@emotion/react": "^11.10.4",
+    "@emotion/styled": "^11.10.4",
+    "gatsby-source-filesystem": "4.24.0",
```

Comment:
Pinning uikit versions to `15.2.4`, as the upgrade to `15.3.0` is tackled in another PR with the design tokens updates #1372